testing/tests/DevExpress.ui.widgets.dataGrid/editing.tests.js

```diff
@@ -5304,7 +5304,7 @@ QUnit.module('Editing with real dataController', {
             isCloseEditCell = true;
         };
 
-        this.editRow(0);
+        this.editCell(0, 0);
```

Comment:
Why?

Reply:
Because `editRow` does not work for batch and cell edit modes. [This is mentioned in our docs](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#editRowrowIndex).